streams/src/test/java/org/apache/kafka/streams/integration/FanoutIntegrationTest.java

```diff
@@ -116,7 +105,6 @@ public void shouldFanoutTheInput() throws Exception {
         streamsConfiguration.put(StreamsConfig.VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
         streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, COMMIT_INTERVAL_MS);
         streamsConfiguration.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-        streamsConfiguration.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, cacheSizeBytes);
```

Comment:
There is no benefit in this test as it doesn't use the cache at all, i.e., only `KTables` use it.